doc/lua_api.txt

```diff
@@ -1556,11 +1577,13 @@ is multiplied by `3^leveldiff`.
 * `uses=10, leveldiff=1`: actual uses: 30
 * `uses=10, leveldiff=2`: actual uses: 90
 
-#### Maximum level
+Maximum level
+-------------
 Tells what is the maximum level of a node of this group that the tool will
 be able to dig.
 
-#### Digging times
+Digging times
```

Comment:
This is actually `###` a level below `Tools definition`as it describes a part of the tool definition.